include/samplog/LogLevel.h

```diff
@@ -14,8 +14,15 @@ enum samplog_LogLevel
 	INFO = 2,
 	WARNING = 4,
 	ERROR = 8,
+	FATAL = 16,
```

Comment:
I've changed the other things, bt IMHO a purely logging framework should not be so opinionated on "you should not have fatal errors, so we will not record them if you do".